starter-code/src/functions-and-arrays.js

```diff
@@ -11,10 +19,27 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord(wordsArray){
```

Comment:
Nope. You are not supposed to hardcode this find longest word function with something like "foo"? 
This function was supposed to find the longest word from a list.

Here is the solution to this part, so you can compare the steps needed:

```js
function findLongestWord(wordsArr) {
  var longest = wordsArr[0];
  var candidate;

  for (var i = 1; i < wordsArr.length; i++) {
    candidate = wordsArr[i];

    if (candidate.length > longest.length) {
      longest = candidate;
    }
  }
  return longest;
}

var longest = findLongestWord(words);
console.log(longest);
```